src/components/Layout/index.js

```diff
@@ -30,7 +30,8 @@ class Layout extends Component {
           <HeaderSection>
             <StyledLink to="/cart">My Cart</StyledLink>|
             <StyledLink to="/account">My Account</StyledLink>|
-            <StyledLink to="/signup">Sign Up</StyledLink>
+            <StyledLink to="/signup">Sign Up</StyledLink>|
```

Comment:
If the user is authenticated you should hide those routes and at the same time display logout route.